plugins/outputs/bigquery/bigquery.go

```diff
@@ -110,7 +110,10 @@ func (b *BigQuery) Write(metrics []telegraf.Metric) error {
 
 	for k, v := range groupedMetrics {
 		wg.Add(1)
-		go b.insertToTable(k, v, &wg)
+		go func(k string, v []bigquery.ValueSaver) {
+			b.insertToTable(k, v)
+			wg.Done()
```

Comment:
Make it `defer wg.Done()` and move it to the top of the anonymous function so we can be sure it will be done in any case.